Tests/PubNubTests/Mocking/MockURLSession.swift

```diff
@@ -133,6 +217,34 @@ class MockURLSession: URLSessionReplaceable {
     return task
   }
 
+  func uploadTask(withStreamedRequest request: URLRequest) -> URLSessionUploadTask {
+    guard let task = self.state.lockedRead({ $0.tasks.first { $0.originalRequest == request } }) as? URLSessionUploadTask else {
+      fatalError("Task not found for matching request \(request)")
+    }
+    return task
+  }
+
+  func uploadTask(with request: URLRequest, fromFile url: URL) -> URLSessionUploadTask {
+    guard let task = self.state.lockedRead({ $0.tasks.first { $0.originalRequest == request } }) as? URLSessionUploadTask else {
+      fatalError("Task not found for matching request \(request)")
+    }
+    return task
+  }
+
+  func downloadTask(with url: URL) -> URLSessionDownloadTask {
+    guard let task = self.state.lockedRead({ $0.tasks.first { $0.originalRequest?.url == url } }) as? URLSessionDownloadTask else {
+      fatalError("Task not found for matching request \(url)")
+    }
+    return task
+  }
+
+  func downloadTask(withResumeData resumeData: Data) -> URLSessionDownloadTask {
+    guard let task = self.state.lockedRead({ $0.tasks.first { ($0 as? MockURLSessionDownloadTask)?.resumeData == resumeData } }) as? URLSessionDownloadTask else {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 162 characters](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=6302911)